app/src/main/res/layout/fragment_splash.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ImageView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:scaleType="fitCenter"
+    android:src="@drawable/ic_basketball"
+    tools:context=".presentation.ui.splash.SplashFragment" />
```

Comment:
![telegram-cloud-photo-size-2-5402444138651169903-y](https://user-images.githubusercontent.com/23121846/136341351-be8a4fdc-c69b-4b3e-bbe0-acfadd8a5dae.jpg)
This is how your screen looks like right now - fix It